.circleci/config.yml

```diff
@@ -70,12 +76,18 @@ workflows:
           filters:
             tags:
               only: /.*/
+      - node-12:
+          # Run for all tags (required to allow the deploy to trigger on version tags)
+          filters:
+            tags:
+              only: /.*/
       - deploy:
           # Deploy passing builds if they're tagged with a version
           requires:
             - node-6
```

Comment:
Maybe we can remove node 6 and 8

Reply:
Yeah we can remove 6, resin-lint still officially supports 8 though so it'd have to be a major to drop it (which I'd like to do, but a major bump should definitely go hand in hand with a rename to balena-lint and would be separate to this patch PR as well)

Reply:
Removed 6